src/applications/vaos/components/VAFacilityLocation.jsx

```diff
@@ -7,7 +7,8 @@ export default function VAFacilityLocation({
   facility,
   facilityName,
   facilityId,
-  isHomepageRefresh,
+  isV2,
```

Comment:
This seems like a bad merge, I think Narin changed `isV2` to `isHomepageRefresh`

Reply:
Fixed.